src/main/java/core/basesyntax/Model/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Model;
+
+import core.basesyntax.Base.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message in the `doWork` method should indicate the type of machine more informatively. Consider including the word 'Truck' in the message to make it clear. For example: 'Bulldozer truck started its work.'